hwdb.d/meson.build

```diff
@@ -43,37 +45,35 @@ if conf.get('ENABLE_HWDB') == 1
                 command : make_autosuspend_rules_py,
                 capture : true,
                 install : true,
-                install_dir: udevhwdbdir)
+                install_dir : udevhwdbdir,
+        )
 
-        install_data(hwdb_files_notest,
-                     hwdb_files_test,
-                     install_dir : udevhwdbdir)
+        install_data(hwdb_files_notest, hwdb_files_test, install_dir : udevhwdbdir)
 
         if install_sysconfdir
-                meson.add_install_script('sh', '-c',
-                                         mkdir_p.format(sysconfdir / 'udev/hwdb.d'))
+                meson.add_install_script('sh', '-c', mkdir_p.format(sysconfdir / 'udev/hwdb.d'))
 
-                meson.add_install_script('sh', '-c',
-                                         'test -n "$DESTDIR" || @0@/systemd-hwdb update'.format(rootbindir))
+                meson.add_install_script(
+                        'sh',
+                        '-c',
+                        'test -n "$DESTDIR" || @0@/systemd-hwdb update'.format(rootbindir),
+                )
         endif
 
         if want_tests != 'false'
                 parse_hwdb_py = find_program('parse_hwdb.py')
-                test('parse-hwdb',
-                     parse_hwdb_py,
-                     suite : 'dist-check',
-                     args : [hwdb_files_test,
-                             auto_suspend_rules],
-                     timeout : 90)
+                test(
+                        'parse-hwdb',
+                        parse_hwdb_py,
+                        suite : 'dist-check',
+                        args : [hwdb_files_test, auto_suspend_rules],
+                        timeout : 90,
+                )
```

Comment:
This is for the worse…

Reply:
> And the nice thing is there are github actions to enforce this, so you'll never get weird styles again.

I find that particularly obnoxious. Incidental contributors who use a different editor and have a different version of the formatting thingymagingy will get a linebreak in a spot and the patch will be "rejected". Enforced formatting raises the bar for contributions. If wouldn't be so bad if the automatic formatting was good. But as the examples here show, quite often it's at best "meh", which means that we enforce something which isn't worth enforcing.

What I'd like to see is a formatter which helps you automatically format the code you edit. I.e.: you edit a piece of code, so you touch it anyway, so it helps you DTRT, but if want to, you can format a piece of code in a different fashion. And it stays in this way until the next person comes along and touches it.